mne/gui/_file_traits.py

```diff
@@ -334,13 +338,24 @@ def _get_inst_fname(self):
     @cached_property
     def _get_inst_points(self):
         if not self.inst:
-            return np.zeros((1, 3))
+            return np.zeros((1, 3))  # why not just None?
```

Comment:
I don't remember, but setting it to None causes an exception on startup, so it would require downstream modification to allow it to be None. Simpler might be setting its shape to (0, 3), that seems to work for me. I don't know why (1, 3).